.github/workflows/access-control-integration-test.yml

```diff
@@ -22,21 +22,32 @@ jobs:
         with:
           filters: |
             source_changes:
+              - .github/**
               - api/**
               - authorizations/**
+              - bin/**
```

Comment:
Why do we need to `bin/**` here? for any changes under folder `bin`, I think `backend test` can cover it.

Reply:
There are some environment and JVM parameters in `bin/common.sh` and `bin/gravitino.sh` that might affect the deployment test?

Reply:
OKay, it's not a big problem any how.